source/frontend/src/features/mapSideBar/acquisition/common/GenerateForm/GenerateFormContainer.tsx

```diff
@@ -14,16 +23,102 @@ export interface IGenerateFormContainerProps {
 const GenerateFormContainer: React.FunctionComponent<
   React.PropsWithChildren<IGenerateFormContainerProps>
 > = ({ acquisitionFileId, View }) => {
-  const generateLetter = useGenerateLetter();
+  const [fullRecipientsList, setFullRecipientsList] = useState<LetterRecipientModel[]>([]);
+  const { getPersonConcept, getOrganizationConcept } = useApiContacts();
+  const {
+    getAcquisitionFile: { execute: getAcquisitionFile },
+  } = useAcquisitionProvider();
+  const [isGenerateLetterModalOpened, openGenerateLetterModal, closeGenerateLetterModal] =
+    useModalManagement();
+  const {
+    getAcquisitionOwners: {
+      execute: retrieveAcquisitionFileOwners,
+      loading: loadingAcquisitionFileOwners,
+    },
+  } = useAcquisitionProvider();
```

Comment:
nit: you already have a previous call to **useAcquisitionProvider()** - please move the getAcquisitionOwners destructuring to the first call and remove this extra call to the hook

Reply:
updated